crates/holochain/src/core/workflow/countersigning_workflow/complete.rs

```diff
@@ -141,6 +142,58 @@ pub(crate) async fn inner_countersigning_session_complete(
         return Ok(None);
     }
 
+    reveal_countersigning_session(
```

Comment:
Extracted function `reveal_countersigning_session` and reused it in `force_publish`. No logic changes.